test/MockAccessToken.js

```diff
@@ -12,7 +12,7 @@ const { TestRedisAdapter, testAccessToken } = require('./testConfig')
 
 const MockAccessToken = (userId) => {
   // get a Redis
-  const adapter = new TestRedisAdapter()
+  const adapter = new TestRedisAdapter('AccessToken')
```

Comment:
Setting the right top level key here allowed us to access the right redis object (field?)